nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -768,6 +768,10 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The `firmwareLinuxNonfree` package has been renamed to `linux-firmware`.
 
+- The `services.tailscale` module now warns that some of its optional
```

Comment:
We don't really need this entry, the changelog is already to long. People that are affected by this will see the warning and things where already half broken for a longer time before.

Reply:
Removed. Honestly I only added it because the PR checklist asked if there were any notable changes that should go in the relnotes.